apps/admin-page/src/app/pages/account/account.component.ts

```diff
@@ -0,0 +1,139 @@
+import { HttpErrorResponse } from '@angular/common/http';
+import { Component } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { MatSnackBar } from '@angular/material/snack-bar';
+import { IUser } from '@interfaces';
+import { UserRoles } from '@models';
+import { UserService } from '../../services/user/user.service';
+import { MatDialog } from '@angular/material/dialog';
+import { ChangePasswordDialogComponent } from '../../components/change-password-dialog/change-password-dialog.component';
+
+@Component({
+  selector: 'webstore-account',
+  templateUrl: './account.component.html',
+  styleUrls: ['./account.component.css'],
+})
+export class AccountComponent {
+  public user?: IUser;
+  public accountForm: FormGroup;
+  public isLoading = false;
+  public isUpdatingInfo = false;
+  public panelOpenState = false;
+
+  /**
+   * Gets the current user.
+   * Initiates the account form.
+   *
+   * @param userService
+   * @param snackBar
+   * @param dialog
+   */
+  constructor(private userService: UserService, private snackBar: MatSnackBar, public dialog: MatDialog) {
+    this.isLoading = true;
+    this.userService.getCurrentUser().subscribe(
+      (user: IUser) => {
+        this.user = user;
+        this.updateForm();
+        this.isLoading = false;
+      },
+      (err: HttpErrorResponse) => {
+        console.log(err.message);
+        this.isLoading = false;
+      }
+    );
+
+    this.accountForm = new FormGroup({
+      name: new FormControl('', [Validators.maxLength(50), Validators.pattern('^[^0-9]+$')]),
+      phoneNumber: new FormControl('', [
+        Validators.minLength(8),
+        Validators.maxLength(11),
```

Comment:
Max phone number length is 15 (I know it doesn't match webstore)

Reply:
Why does this need to be? The current regex allows for a number 12345678 as a min, or +4512345678